skyvern/core/script_generations/generate_script.py

```
@@ -1781,84 +1822,67 @@ async def generate_workflow_script_python_code(
             except Exception as e:
                 LOG.error("Failed to create script block", error=str(e), exc_info=True)
-                # Continue without script block creation if it fails
 
-        block_fns.append(block_fn_def)
-        if idx < len(task_v1_blocks) - 1:
-            block_fns.append(cst.EmptyLine())
-            block_fns.append(cst.EmptyLine())
+        append_block_code(block_code)
 
     # Handle task_v2 blocks
     for idx, task_v2 in enumerate(task_v2_blocks):
```

Ruff (0.14.3)

Loop control variable `idx` not used within loop body

(B007)